src/routes/card-payment/payment.css

```diff
@@ -1,164 +1,164 @@
 .dFlex {
-  display: flex;
+    display: flex;
```

Comment:
There shouldnt be any changes to payment.css. also if you have prettier, please use alt shift f to fix this.